### DISCUSSION TITLE
Making activity private should not hide pinned repositories

### DISCUSSION THREAD

**Discussion by aschier** [community/community#37354] [Other Feature Feedback, Questions, & Ideas]
When choosing to make the activity private, the pinned repositories should stay visible. They are pinned for others to see, if one does not want this, one can unpin them.

In addition I think it would be better to just hide the activity tiles instead of showing prominently "activity is private". There is no point in showing the area at all, if it is disabled.

**Comment by ossanna16** [comment]
Hi there @aschier and welcome to our community! Thank you for the suggestion 🙂

**Comment by martiliones** [comment]
It makes sense because you can see all user's public repositories by switching to another tab
